cmd/system-probe/modules/eventmonitor.go

```diff
@@ -35,6 +35,7 @@ func createEventMonitorModule(_ *sysconfigtypes.Config, deps module.FactoryDepen
 	}
 
 	opts := eventmonitor.Opts{}
+	opts.ProbeOpts.Tagger = deps.Tagger
```

Comment:
let's not do this for now, this will most likely what we will end up doing, but I would prefer if this is done in the same PR that also remove the downstream init of the remote tagger we currently have

Reply:
Alright. This means all the code for adding the tagger to the factory dependencies is not technically needed right now. Do you want me to keep it still, or should I remove it as well?

Reply:
keeping the rest is fine to me

Reply:
Got it, I'll just remove that one line then.

Reply:
@paulcacheux out of curiosity. Are there any other work needed to be done for removing downstream init of the remote tagger? Mostly, to understand why we do not do it as part of this PR 😄

Reply:
There is no other dependency, it's just that you need to fix some plumbing in our code to make sure the tests and all that work out of the box. This PR was passing the tests by chance because it was passing the tagger similarly to how we pass the no-op tagger in our tests, this is not really something we want to keep. Once this is merged I will open a PR fixing all the CWS code to use this